vars/neoDeploy.groovy

```diff
@@ -84,10 +84,15 @@ import static com.sap.piper.Prerequisites.checkScript
      * @see dockerExecute
      */
     'dockerOptions',
+    /**
+      * Extension files. Provided to the neo command via parameter `--extensions` (`-e`).
+      */
+    'mtaExtensionDescriptors',
     /**
      * The path to the archive for deployment to SAP CP. If not provided `mtarFilePath` from commom pipeline environment is used instead.
      */
-    'source'
+    'source',
```

Comment:
This , is not needed

Reply:
see above

Reply:
> [parameter after "s"] the probability [...]

I did some research the frequency distribution of characters between `t` and `z` (including both boundaries) in english is 18%. Of course that is the overall frequency. I did not take into account the frequency of first-characters. So maybe there is a bias.

How do you define _very low_?.

Reply:
And I forgot: I guess I added the `,` during the development process having the parameter somehow named differently ( ... I forgot the details). Afterwards I shifted the parameter upwards and left the comma by mistake. Intentionally I would never add a comma here (as standalone change) since this blurs the focus of the PR ... :-)

Reply:
18% is low :) I think you did move it because Christopher told you to keep the order. So, yes, it blurs the focus of the PR. The answer was even more simple :)